app/views/admin/users/index.html.erb

```diff
@@ -1,33 +1,73 @@
-<header>
-  <h1 class="crayons-title">People</h1>
-  <%= render "menu" %>
-</header>
+<% if FeatureFlag.enabled?(:member_index_view) %>
+  <div class="crayons-card p-3 s:p-4 m:p-7">
+    <header class="flex justify-content-between">
+      <h1 class="crayons-title">Members</h1>
+      <%= render "admin/users/index/tabs" %>
+    </header>
 
-<%= paginate @users, theme: "internal" %>
+    <%= render "admin/users/index/controls" %>
 
-<table class="crayons-table" width="100%">
-  <thead>
-    <tr>
-      <th scope="col">Username</th>
-      <th scope="col">ID</th>
-      <th scope="col">Name</th>
-      <th scope="col">Twitter</th>
-      <th scope="col">GitHub</th>
-      <th scope="col">Email</th>
-    </tr>
-  </thead>
-  <tbody class="crayons-card">
-    <% @users.each do |user| %>
-    <tr>
-      <td><%= link_to user.username, admin_user_path(user) %></td>
-      <td class="whitespace-nowrap"><%= user.id %></td>
-      <td><%= user.name %></td>
-      <td><%= user_twitter_link(user) %></td>
-      <td><%= user_github_link(user) %></td>
-      <td><%= user.email %></td>
-    </tr>
-    <% end %>
-  </tbody>
-</table>
+    <%= paginate @users, theme: "internal" %>
 
-<%= paginate @users, theme: "internal" %>
+      <table class="crayons-table" width="100%">
+        <thead>
+          <tr>
+            <th scope="col">Username</th>
+            <th scope="col">ID</th>
+            <th scope="col">Name</th>
+            <th scope="col">Twitter</th>
+            <th scope="col">GitHub</th>
+            <th scope="col">Email</th>
+          </tr>
+        </thead>
+        <tbody class="crayons-card">
+          <% @users.each do |user| %>
+          <tr>
+            <td><%= link_to user.username, admin_user_path(user) %></td>
+            <td class="whitespace-nowrap"><%= user.id %></td>
+            <td><%= user.name %></td>
+            <td><%= user_twitter_link(user) %></td>
+            <td><%= user_github_link(user) %></td>
+            <td><%= user.email %></td>
+          </tr>
+          <% end %>
+        </tbody>
+      </table>
+
+      <%= paginate @users, theme: "internal" %>
+  </div>
+<% else %>
+  <header>
+    <h1 class="crayons-title">People</h1>
+    <%= render "menu" %>
```

Comment:
Eventually we'll remove everything in the else block including this partial. I've created two new partials, tabs and filters and moved over the code for the new view.